src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Remote/RemoteServiceInvoker.cs

```diff
@@ -105,16 +103,25 @@ internal sealed class RemoteServiceInvoker(
         // so we get the regular (MessagePack) client too.
```

Comment:
Isn't this comment simply not true any more, so we can just delete this little "get both clients just in case" entirely?

Reply:
that was my question to you - the comment made it sound like you always had to get MessagePackage client before getting JSON client. I am guessing it's not the case. I'll delete the comment and will simplify the code, which will also address Dustin's feedback.